model_analyzer/result/result_table_manager.py

```diff
@@ -0,0 +1,609 @@
+
+        Parameters
+        ----------
+        writer : OutputWriter
+            Used to write the result tables to an output stream
+        column_separator : str
+            The string that will be inserted between each column
+            of the table
+
+        Raises
+        ------
+        TritonModelAnalyzerException
+        """
+
+        for table in self._result_tables.values():
+            self._write_result(table,
+                               writer,
+                               column_separator,
+                               ignore_widths=False)
+
+    def _write_result(self,
+                      table,
+                      writer,
+                      column_separator,
+                      ignore_widths=False,
+                      include_title=True):
+        """
+        Utility function that writes any table
+        """
+
+        if include_title:
+            writer.write('\n'.join([
+                table.title() + ":",
+                table.to_formatted_string(separator=column_separator,
+                                          ignore_widths=ignore_widths), "\n"
+            ]))
+        else:
+            writer.write(
+                table.to_formatted_string(separator=column_separator,
+                                          ignore_widths=ignore_widths) + "\n\n")
+
+    def _add_server_data(self):
+        """
+        Adds data to directly to the server only table
+
+        Parameters
+        ----------
+        data : dict
+            keys are gpu ids and values are lists of metric values
+        """
+
+        server_fields = self._server_output_fields
+        server_only_data = self._result_manager.get_server_only_data()
+
+        for gpu_uuid, metrics in server_only_data.items():
+            data_row = [None] * len(server_fields)
+
+            model_name_index = self._find_index_for_field(
+                server_fields, 'model_name')
+            if model_name_index is not None:
+                data_row[model_name_index] = 'triton-server'
+
+            gpu_uuid_index = self._find_index_for_field(server_fields,
+                                                        'gpu_uuid')
+            if gpu_uuid_index is not None:
+                data_row[gpu_uuid_index] = gpu_uuid
+
+            for metric in metrics:
+                metric_tag_index = self._find_index_for_field(
+                    server_fields, metric.tag)
+
+                if metric_tag_index is not None:
+                    data_row[metric_tag_index] = round(metric.value(), 1)
+            self._result_tables[self.server_only_table_key].insert_row_by_index(
+                data_row)
+
+    def _add_result_table(self, table_key, title, headers):
+        """
+        Utility function that creates a table with column
+        headers corresponding to perf_analyzer arguments
+        and requested metrics. Also sets the result
+        comparator for that table.
+        """
+
+        # Create headers
```

Comment:
Is this comment correct?
This line creates the whole table including _headers_ right?

Reply:
it would appear so. Removing the comment